{{ cookiecutter.app_name }}/{{ cookiecutter.project_name }}/settings.py

```diff
@@ -0,0 +1,146 @@
+"""
+Django settings for {{ cookiecutter.app_name }} pp. This is just a harness type
+project for testing and interacting with the app.
+
+
+For more information on this file, see
+https://docs.djangoproject.com/en/1.8/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/1.8/ref/settings/
+
+"""
+import os
+
+import dj_database_url
+import yaml
+
+CONFIG_PATHS = [
+    os.environ.get('{{ cookiecutter.project_name|upper }}_CONFIG', ''),
+    os.path.join(os.getcwd(), '{{ cookiecutter.project_name|lower }}.yml'),
+    os.path.join(os.path.expanduser('~'), '{{ cookiecutter.project_name|lower }}.yml'),
+    '/etc/{{ cookiecutter.project_name|lower }}.yml',
+]
+
+
+def load_fallback():
+    """Load optional yaml config"""
+    fallback_config = {}
+    config_file_path = None
```

Comment:
Minor but clearer: Either change the default value of `config_file_path` to `""`, or below check for `if config_file_path is not None`. I prefer `""`, because that way the variable is the same type at all times.